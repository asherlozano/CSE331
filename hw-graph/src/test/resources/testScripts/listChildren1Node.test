# listChildren1Node test file
# tests if you can listChildren method can be added to graph
# listChildren lists all the children of a node

# Create a graph
CreateGraph g1

# Create a node
AddNode g1 n1

# Create an edge
AddEdge g1 n1 n1 e1

# Print the graphs nodes
ListNodes g1
ListChildren g1 n1