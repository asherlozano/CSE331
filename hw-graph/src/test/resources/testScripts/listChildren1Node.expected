# listChildren1Node expected file
# tests if the listChildren method can be added to graph
# listChildren lists all the children of a node

# Create a graph
created graph g1

# Create a node
added node n1 to g1

# Create an edge
added edge e1 from n1 to n1 in g1

# Print the nodes in the graph
g1 contains: n1
the children of n1 in g1 are: n1(e1)